core/arch/arm/plat-versal/main.c

```diff
@@ -0,0 +1,61 @@
+// SPDX-License-Identifier: BSD-2-Clause
+/*
+ * Copyright (C) Foundries Ltd. 2022 - All Rights Reserved
+ */
+
+#include <platform_config.h>
+
+#include <stdint.h>
+#include <string.h>
+
+#include <drivers/gic.h>
+#include <drivers/pl011.h>
+
+#include <arm.h>
+#include <console.h>
+#include <kernel/boot.h>
+#include <kernel/interrupt.h>
+#include <kernel/misc.h>
+#include <kernel/tee_time.h>
+#include <mm/core_memprot.h>
+#include <trace.h>
```

Comment:
Minor: prefer all includes in one block and in alphabetical order

Reply:
ah sure, sorry I wasnt sure if this rule had been changed recently. I remember seeing some crypto commit (crypto/aspeed) with all the includes unsorted. will fix.